data/presets/office/courier.json

```diff
@@ -0,0 +1,32 @@
+{
+    "icon": "fas-people-carry-box",
+    "fields": [
+        "name",
+        "address",
+        "opening_hours"
+    ],
+    "moreFields": [
+        "{@templates/poi}",
+        "air_conditioning",
+        "branch_brand",
+        "brand"
```

Comment:
I'd say, for courier services, the _brand_ information is quite important. Maybe we should promote at least the `brand` field to a regular `field`?

Reply:
I agree with you! I'll fix that.